config/routes.yaml

```diff
@@ -0,0 +1,11 @@
+bgm_ajax_get_address_predictions:
+    path: /ajax/address/get-predictions
+    controller: eckinox_address.controller.address::getAddressPredictions
+    options:
+        expose: true
+
+bgm_ajax_get_address_details:
```

Comment:
Noms de routes contiennent encore `bgm` ;D

Reply:
oh no

Fixed it !

Reply:
Bon point !